lib/particular-calendars/canada.ts

```diff
@@ -1,88 +1,119 @@
 import { PatronTitles } from '../constants/martyrology-metadata';
 import { Precedences } from '../constants/precedences';
 import { CalendarDef } from '../models/calendar-def';
-import { Inputs } from '../types/calendar-def';
+import { Inputs, ParticularConfig } from '../types/calendar-def';
 import { Americas } from './americas';
 
 export class Canada extends CalendarDef {
   parentCalendar = Americas;
 
+  particularConfig: ParticularConfig = {
+    epiphanyOnSunday: true,
```

Comment:
We can use [this](www.cccb.ca/wp-content/uploads/2023/09/2023-2024.pdf#page=2) as a source for this.

Reply:
For this celebration, the missal explicitly states 'Sunday after the first of January,' so I should add `// src: mr_fr_2021_ed3` here.

Reply:
Okay, I didn’t read/translate all those rubrics in the Missal (I am not that good at French, I need to use Google Translate in order to be sure I get the details right and that takes some time :wink:).

Reply:
No problem 😉, in fact, I should have added this comment for Epiphany already.